cmd/environments/get_test.go

```diff
@@ -138,8 +139,46 @@ func TestGet(t *testing.T) {
 			"--project", "test-proj",
 		}
 
-		_, err := cmd.CallCmd(t, clients, args)
+		_, err := cmd.CallCmd(t, clients, &analytics.NoopClient{}, args)
 
 		assert.EqualError(t, err, "base-uri is invalid"+errorHelp)
 	})
+
+	t.Run("will track analytics for 'CLI Command Run' event", func(t *testing.T) {
+		id := "test-id"
+		mockedTrackingArgs := []interface{}{
```

Comment:
You could remove some duplication by extracting this to a helper function.
```
// analytics/client.go
func MockedTracker(name string, action string, flags []string) *MockTracker {
	id := "test-id"
	mockedTrackingArgs := []interface{}{
		"testAccessToken",
		"http://test.com",
		"CLI Command Run",
		map[string]interface{}{
			"action":  action,
			"baseURI": "http://test.com",
			"flags":   flags,
			"id":      id,
			"name":    name,
		},
	}
	tracker := MockTracker{ID: id}
	tracker.On("SendEvent", mockedTrackingArgs...)

	return &tracker
}
```
and call it in the tests like
```
tracker := analytics.MockedTracker(
    "flags",
    "get",
    []string{"access-token", "base-uri", "environment", "flag", "project"}, // these need to be alphabetized
)
```